### ISSUE TITLE
bug: (mobile) incorrect cropping of images

### ISSUE THREAD

**Issue by ankush** [The-Commit-Company/Raven#834]
![Screenshot_2024-04-10-00-26-53-79_3aea4af51f236e4932235fdada7d1643](https://github.com/The-Commit-Company/Raven/assets/9079960/1775071e-8515-44af-be11-9d7e3c999962)

**Comment by ankush** [comment]
Raw image if required

![image2261bc](https://github.com/The-Commit-Company/Raven/assets/9079960/0631ff2a-b4ef-473c-94fc-6d011fe95506)

**Comment by ankush** [comment]
IMO barring extreme aspect ratios we shouldn't crop images almost ever (?)

**Comment by nikkothari22** [comment]
> IMO barring extreme aspect ratios we shouldn't crop images almost ever (?)

Yep. Seems like a recent bug via #769. Should be an easy fix though.
